sksurv/ensemble/boosting.py

```diff
@@ -375,7 +375,8 @@ def _make_estimator(self, append=True, random_state=None):
 
 class GradientBoostingSurvivalAnalysis(BaseGradientBoosting, SurvivalAnalysisMixin):
     r"""Gradient-boosted Cox proportional hazard loss with
-    regression trees as base learner.
+    regression trees as base learner.  In each stage, ``n_classes_`` regression trees 
```

Comment:
Only a single regression tree is fit in each round, not `n_classes_`.
Also, add a blank line after the first sentence and remove trailing whitespace.